build_automation/frontend/src/js/tag_management.js

```diff
@@ -250,20 +320,47 @@ class TagManagementComponent extends React.Component {
 
                     <Grid container spacing={0}>
                         <Grid item xs={12}>
-                            <ExpansionPanel expanded={expanded === 'coverage'} onChange={this.handleChange('coverage')} onClick={e => { this.handleAccordionClick('coverage') }}>
-                                <ExpansionPanelSummary expandIcon={<ExpandMoreIcon />} onClick={e => { this.setUrls(APP_URLS.COVERAGES_LIST, APP_URLS.COVERAGES_DETAIL) }}>
-                                    <Typography>Coverages</Typography>
+                            <ExpansionPanel expanded={expanded === 'creator'} onChange={this.handleChange('creator')} onClick={e => { this.handleAccordionClick('creator') }}>
+                                <ExpansionPanelSummary expandIcon={<ExpandMoreIcon />} onClick={e => { this.setUrls(APP_URLS.CREATORS_LIST, APP_URLS.CREATORS_DETAIL) }}>
+                                    <Typography>Creators</Typography>
                                 </ExpansionPanelSummary>
                                 <ExpansionPanelDetails>
                                     <Grid container>
                                         <Grid item>
-                                            <Button variant="raised" color="primary" onClick={e => { this.setCurrentView('addTag', 'Coverages') }}>
-                                                Add New
-            </Button>
+
                                         </Grid>
                                         <DataGrid
-                                            rows={coverageRows}
-                                            columns={coverageColumns}
+                                            rows={creatorRows}
+                                            columns={creatorColumns}
+                                        >
+                                            <FilteringState defaultFilters={[]} columnExtensions={[{ columnName: 'name', filteringEnabled: true }]} />
+                                            <IntegratedFiltering />
+                                            <PagingState defaultCurrentPage={0} defaultPageSize={10} />
+                                            <IntegratedPaging />
+                                            <Table rowComponent={obj => { return this.tableRowComponent(obj, 'selectedTagsMenu') }} />
+                                            <TableHeaderRow />
+                                            <TableColumnVisibility />
```

Comment:
`TableColumnVisibility` is not needed, since there are only two columns. And so is `Toolbar`.